xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -813,9 +814,44 @@ public static string PemEncodeSigningRequest(CertificateRequest request, PkcsSig
         <param name="signatureGenerator">The signature generator with which to sign the request.</param>
         <summary>Creates an ASN.1 DER-encoded PKCS#10 CertificationRequest representing the current state of the current object using the provided signature generator.</summary>
         <returns>A DER-encoded certificate signing request.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+When submitting a certificate signing request via a web browser, or other graphical or textual
+              interface, the input is frequently expected to be in the PEM (Privacy Enhanced Mail) format,
+              instead of the DER binary format.
+
+          ]]></format>
+        </remarks>
         <exception cref="T:System.ArgumentNullException">
           <paramref name="signatureGenerator" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.InvalidOperationException">
+          <see cref="P:System.Security.Cryptography.X509Certificates.CertificateRequest.OtherRequestAttributes" /> contains a <see langword="null" /> value.
```

Comment:
I think this member still isn't present on the docs site, so you'll need to back out this `<remarks>` (or the whole file) again.

Reply:
Ah, right. Thanks for the reminder. I'll remove the remarks.